src/controllers/user.js

```diff
@@ -37,15 +37,15 @@ export const getById = async (req, res) => {
   try {
     let foundUser = {}
     const isTeacher = 'TEACHER'
```

Comment:
the role will need to be checked here, the if statement will always return false currently